### DISCUSSION TITLE
Command to return to bed

### DISCUSSION THREAD

**Discussion by AdmiralSky** [Zrips/CMI#6174] [Enhancement]
CMI has the ability to execute commands when you lie down in bed or when you wake up.
However, just right-clicking on the bed will not trigger the command.
It would be great if you could provide a command to move to the bed that is the respawn point, like in Essentials.
It would also be nice if you could add an event when you right-click the bed.

**Comment by CloudeLecaw** [comment]
U can define that homepoint will be set when move into bed.
```

  # Requires cmi.bedhome permission node
  # When set to true by interacting with bed, regular home location will be created
  # When set to false vanilla bed home location will be set
  BedInteraction: false
  # When set to true, you can only set home location when you shift click a bed
  onlyShiftBed: false
  # Removes bed location when home location was created by clicking on that bed
  RemoveBedLocationOnBedBreak: true
  # When enabled, plugin will check if player can break block in that area before creating new home location
  # This allows to prevent setting homes in protected areas
  # Can be bypassed with cmi.command.sethome.bypass
  CheckBlockBreak: true
  # Name for default home location. By default its "Home"
  DefaultName: Home
  # Name for bed location. By default its "Home"
  # Different name can be used to identify home location set by interacting with a bed from home location created with command
  DefaultBedName: Home
```
You can get there thennwith using the home command

**Comment by Zrips** [comment]
We do have respawn priority list which will respawn you on your bed location if possible. Additionally you can automatically create home location by interacting with bed which would allow you to return to bed location with home location. 
What exact behavior are you missing here?
